### DISCUSSION TITLE
Wrong APK file name?

### DISCUSSION THREAD

**Discussion by devfbe** [goreleaser/nfpm#790] [Q&A]
Hey there,

I'm currently trying to package apk files for alpine linux.

When calling `nfpm package -p apk -t build`, nfpm generates a package name like

`mypkg_1.0.0_rc0_x86_64.apk`

The alpine upstream repositories don't contain an arch in the filename and there are only hyphens instead of underscores in the "official" alpine packages. So is there any reason why nfpm uses underscores and includes the arch in the apk file name (or is that simply a bug and it's worth opening an issue?)

Thanks :-)

**Comment by caarlos0** [comment]
at the time it was implemented, I think there weren't any guidelines of how the files should be named... 

if it's incorrect now, we can change it, yes.
